using-timescaledb/telemetry.md

```diff
@@ -0,0 +1,52 @@
+# Telemetry and Version Checking
+Starting in v0.12.0 we enable anonymous usage sharing to help us better understand and assist Timescale users. We emphasize that privacy of our users is paramount, so we do not collect any identifying information. The following is an example of the JSON that is sent to our servers about a specific deployment:
+```
+{
+	“db_uuid”: “26917841-2fc0-48fd-b096-ba19b3fda98f”,
+	“exported_db_uuid”: “8dd4543c-f44e-43c9-a666-02d23bb09b90”,
+	“installed_time”: “2000-04-17 10:56:59.427738-04”,
+	“install_method”:”source”,
+	“os_name”:”Darwin”,
+	“os_version”:”Darwin Kernel Version 17.5.0: Mon Mar  5 22:24:32 PST 2018”,
+	“os_release”:”17.5.0”,
+	“postgresql_version”:”10.4”,
+	“timescaledb_version”:”0.12.0”,
+	“build_os_name”:”Darwin”,
+	“build_os_version”:”17.5.0”,
+	“data_volume”: “65982148”,
+	“num_hypertables”: “3”,
+	“related_extensions”: 
+    	{
+		“pg_prometheus”:false,
+		“PostGIS”:true
+    	}
+}
+```
+Specifically, both UUID fields are randomly generated by appropriately seeded random
+number generators, so there is no identification information in this field. For full
+transparency, we expose a new API function, [`get_report`], that prints out the exact JSON that is sent to our servers.
+
+## Version Checking
+In response to the telemetry report, users will automatically receive the most recent version of Timescale available for installation. This version will be recorded in the user’s Timescale logs, along with any applicable out-of-date version warnings. While you do not have to update immediately to the newest release, many users have reported that performance issues or bugs automatically resolve after updating their version of Timescale. 
```

Comment:
I "think" we want to say TimescaleDB here

Reply:
Yeah fixed.